### ISSUE TITLE
Cannot set generic List for field type

### ISSUE THREAD

**Issue by antoniodiego** [argouml-tigris-org/argouml#63]

**Comment by linustolke** [comment]
Yes, I don't think the java profile used is modern enough to include generics. This is an old project. Some major additions is probably needed, both to the UML and to the Java profile to support generics.
